vendor/saft/lm17500.yaml

```diff
@@ -0,0 +1,29 @@
+name: LM 17500 Primary Li-MnO2 cell - 3 V lithium manganese dioxide A-size spiral cell
+description: Saft's LM 17500 cell is ideally suited for applications requiring high energy and long operating life, with stable voltage under high discharge rates in -40°C / + 85°C environment.
+
```

Comment:
Hi @raenaka it is necessary to add hardware and firmware version for validation, if there is no, as it's a battery you can add it `version: '1.0'`